tests/test_webhooks.py

```diff
@@ -67,9 +67,23 @@ def test_timestamp_outside_threshold(
             self.webhooks.verify_event(mock_event_body, mock_header, mock_secret, 0)
         assert "Timestamp outside the tolerance zone" in str(err.value)
 
-    def test_sig_hash_matches_expected_sig(self, mock_sig_hash):
+    def test_sig_hash_does_not_match_expected_sig_length(self, mock_sig_hash):
         with pytest.raises(ValueError) as err:
-            self.webhooks.constant_time_compare(mock_sig_hash, "q234q23r23423")
+            self.webhooks.constant_time_compare(
+                mock_sig_hash,
+                "df25b6efdd39d82e7b30e75ea19655b306860ad5cde3eeaeb6f1dfea029ea25",
+            )
+        assert (
+            "Signature hash does not match the expected signature hash for payload"
+            in str(err.value)
+        )
+
+    def test_sig_hash_does_not_match_expected_sig_value(self, mock_sig_hash):
+        with pytest.raises(ValueError) as err:
+            self.webhooks.constant_time_compare(
+                mock_sig_hash,
+                "df25b6efdd39d82e7b30e75ea19655b306860ad5cde3eeaeb6f1dfea029ea252",
+            )
```

Comment:
It seems the test logic is duplicated. Could you check?

Reply:
Hey Jonatas, thanks for reviewing! These tests are actually testing for 2 different things. The first tests if the error will be thrown based on the length of the signature hash, and the second tests if the error will be thrown if the signature hash is the same length, but has a different value. Prior to this PR, the second test would have failed.

Reply:
I understand the purpose is different, but the code is exactly the same. Not sure if I'm missing something.

Reply:
Oh.. Now I see that `2` is appended in the second test.